homeassistant/components/homeassistant_hardware/strings.json

```diff
@@ -18,18 +18,29 @@
             "uninstall_addon": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::step::uninstall_addon::title%]"
           }
         },
+        "change_channel": {
+          "title": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::step::reconfigure_addon::title%]",
+          "data": {
+            "channel": "Channel"
+          }
+        },
         "install_addon": {
           "title": "The Silicon Labs Multiprotocol add-on installation has started"
         },
         "notify_channel_change": {
           "title": "Channel change initiated",
           "description": "A Zigbee and Thread channel change has been initiated and will finish in {delay_minutes} minutes."
         },
+        "notify_no_multipan_user": {
+          "title": "Manual configuration may be needed",
+          "description": "Home Assistant found no integrations using the IEEE 802.15.4 radio. If you have configured an integration other than {known_platforms} to use the radio yourself, for example Zigbee2MQTT, you will have to reconfigure the channel in that integration after completing this guide."
+        },
+        "notify_unknown_multipan_user": {
+          "title": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::step::notify_no_multipan_user::title%]",
+          "description": "Home Assistant found integrations {active_platforms} using the IEEE 802.15.4 radio, expected to find all of {known_platforms}. If you have configured an integration to use the radio yourself, for example Zigbee2MQTT, you will have to reconfigure the channel in that integration after completing this guide."
```

Comment:
Maybe it's already explained somewhere else, but this message doesn't really explain why less found platforms than expected is a problem and how that is related to needing to reconfigure the channel of any other integration.

The symptom of fewer platforms seems more like a technical detail that we as developers use as a sign, but that may not really be interesting for the user?